doc/docs/openapi.md

```diff
@@ -130,6 +130,44 @@ import sttp.apispec.openapi.circe._
 println(Printer.spaces2.print(docs.asJson))
 ```
 
+### Support for OpenAPI 3.1.0
+
+Generating OpenAPI documentation compatible with 3.1.0 specifications is a matter of using a different encoder.
+For example, generating the OpenAPI 3.1.0 YAML string can be achieved by performing the following steps:
+
+Firstly add dependencies:
+```scala
+"com.softwaremill.sttp.tapir" %% "tapir-openapi-docs" % "@VERSION@"
+"com.softwaremill.sttp.apispec" %% "openapi-circe-yaml" % " // see https://github.com/softwaremill/sttp-apispec
+```
+
+Create an auxiliary method for encoding OpenAPI object to OpenAPI 3.1 YAML String:
+```scala mdoc:compile-only
+import io.circe.syntax.EncoderOps
+import io.circe.yaml.Printer
+import sttp.apispec.openapi.OpenAPI
+import sttp.apispec.openapi.circe.SttpOpenAPI3_1CirceEncoders
+
+object Encoding extends SttpOpenAPI3_1CirceEncoders { // using `SttpOpenAPI3_1CirceEncoders` ensures that proper encoder would be selected
```

Comment:
maybe we should have such an object in sttp-apispec so that it can simply be used?

Reply:
that is, for `SttpOpenAPICirceEncoders` we already have an implementing object: `sttp.apispec.openapi.circe`. We could similarly have a ready-to-use `sttp.apispec.openapi.circe. SttpOpenAPI3_1CirceEncoders` object

Reply:
updated, `sttp-apispec` introduced `toYaml3_1` extension method, so that generating YAML is a matter of calling it 
```scala
import sttp.apispec.openapi.circe.yaml._ // for `toYaml3_1` extension method

val docs: OpenAPI = ???
println(docs.toYaml3_1) // YAML printed to console
```